configure.ac

```diff
@@ -990,7 +990,7 @@ AS_IF([test -z "$mkmaindll"], [mkmaindll=$mksharedlib])
 
 natdynlink=false
 
-AS_IF([test x"$enable_shared" != "xno"],
+AS_IF([test x"$shared_libraries_supported" = 'xtrue'],
```

Comment:
Belt-and-braces - `natdynlink` should not be set to `true` if we just determined that shared libraries aren't supported